source/documentation/common/storage/proc-Adding_Local_Storage.adoc

```diff
@@ -13,5 +13,14 @@ Adding local storage to a host places the host in a new data center and cluster.
 . Set the path to your local storage in the text entry field.
```

Comment:
Suggestion: Start this sentence with the location unless it is obvious (For example, "In the XYZ manager, ..."

Reply:
@RichardHoch I had the same comment when I first started and it wasn't obvious to me. But then I realized that it should be obvious. This is the Administration Guide so it's talking about the Administration Portal. Most or all of the other procedures don't state it.